pulp_container/app/registry.py

```diff
@@ -64,17 +65,15 @@ async def _dispatch(file, headers):
             The :class:`aiohttp.web.FileResponse` for the file.
 
         """
+        path = os.path.join(settings.MEDIA_ROOT, file.name)
+        if not os.path.exists(path):
+            raise Exception(_("Expected path '{}' is not found").format(path))
```

Comment:
```suggestion
            raise Exception(_("Expected path '{path}' is not found").format(path=path))
```

Reply:
Is this the message that will be sent to the http client?
Should we say something like "The requested artifact '{path}' is missing in the backend storage."

Also, from a security viewpoint, it may be undesired to expose any file system information to the client.
@ggainey ?

Reply:
Nginx does not forward detailed error messages when an internal server error occurs. So, the path should be visible only for administrators, I believe.

Reply:
that's correct @lubosmj

Reply:
Nice.